apps/mdn/mdn-aws/k8s/regions/germany/prod.mm.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+echo '--> Setting environment to STANDBY in GERMANY'
+
+export KUBECONFIG=${HOME}/.kube/germany.config
+
+# Define defaults for environment variables that personalize the commands.
+export TARGET_ENVIRONMENT=prod
+export K8S_NAMESPACE=mdn-${TARGET_ENVIRONMENT}
+export AWS_REGION=eu-central-1
+export K8S_CLUSTER_SHORT_NAME=germany
```

Comment:
Are you calling this Germany and not Frankfurt?

Reply:
We're using a different name to distinguish this from MozMEAO's `frankfurt`.